release-handbook/major_release_workflow.md

```diff
@@ -0,0 +1,184 @@
+            ```
+
+    - Open the `versions.json` file. Confirm that the previous release version number was added to the top of the list. (This is typically done automatically by the command issued in Step 3.) If not, manually add the version number. For example:
+
+        ```
+        [
+        "v3.5.x",
+        "v3.4.x",
+        "v3.3.x",
+        "v3.2.x",
+        "v3.1.x",
+        "v3.0.x"
+        ]
+        ```
+
+5. Create placeholder files to prep for the release notes and TPSRs:
+    - In Visual Studio Code, add the release notes placeholder file for the new major release:
+        1. Go to `/docs/whats-new/release-notes`.
+        2. Create a file for the new version and add the outline to the document. For example: `/docs/whats-new/release-notes/v4_0_0.md`
+
+            To insert the outline, copy and paste the template from the [Release Notes guide](release_notes_guide.md). Ensure that you update the date and release version numbers included in the template.
+
+        3. Open the `sidebars.js` file and add the new page to the release notes section.
+            ```
+            {
+            type: "category",
+            label: "Release notes",
+            items: [
+                "whats-new/release-notes/v4_0_0",
+                "whats-new/release-notes/v3_5_0",
+                "whats-new/release-notes/v3_4_0",
+            ],
+            },
+            ```
+
+    - Add the TPSR placeholder file.
+        1. Go to the `/tpsr` directory.
+        2. Add a new file for the new version. For example: `/tpsr/tpsr-v4.0.x.md`.
+        3. Update the TPSR file URL the `sidebar.js` file:
+
+            ```
+            {
+            type: "link",
+            label: "Third Party Software Requirements",
+            href: "https://github.com/zowe/docs-site/tree/master/tpsr/tpsr-v4.0.x.md",
+            },
+            ```
+
+              **Note:** This is just a placeholder file. The day before GA, this file should be updated to include the information for the new release. Contact the Zowe release engineer to get the most up-to-date file.
+
+    - Create a local build to confirm everything works in your temporary branch.
+
+        1. Run `npm install`.
+        2. Run `npm start` to build the site locally and clear any errors.
+6. Create a PR in the `docs-site` repo to merge the new branch to the `docs-staging` branch. Set the PR to draft mode.
+    - At this point, the branch becomes available to Zowe community members for them to make any needed doc updates. You should now decide what restrictions (if any) to put in place to protect the branch.
+7. As the GA date for the major release approaches, continue to sync the new branch with `docs-staging`. Resolve any merge conflicts that show up. This might mean checking in with the Doc Squad to confirm which content is the correct content to merge.
+8. The day or two before the GA date, perform Step 3 to include the last minor release of the previous major version.
```

Comment:
Should we clarify that this is the point that no new changes should be merged into `docs-staging` (or `master`) to avoid complicated last-minute conflicts?

Reply:
@zFernand0  So this step is to include the last minor release in the major release staging branch so that the Zowe Docs drop-down version menu includes it. I think folks can still make edits to the docs-staging and/or master branches if needed.